commit 1d1e96c094570c3a822f835abbff0a5bca4ca10e
Author: Daniel Kolesa <daniel@octaforge.org>
Date:   Sun Mar 19 19:11:00 2023 +0100

    Disable 9bc89b821cf09195418f20225bbec39c0567d263
    
    Even if we manually define _DYNAMIC, it does not work (segfaults
    somewhere in musl, coming from g_strsplit) and is pretty useless
    for our case anyway.

diff --git a/src/main.c b/src/main.c
index 8cccc0a..ef6408f 100644
--- a/src/main.c
+++ b/src/main.c
@@ -19,6 +19,10 @@
 #include <atk-bridge.h>
 #include <link.h>
 
+#ifdef HAVE_EXE_INTROSPECTION
+#undef HAVE_EXE_INTROSPECTION
+#endif
+
 #ifdef HAVE_EXE_INTROSPECTION
 #include <elf.h>
 #endif
